README.md

```diff
@@ -4,27 +4,6 @@
 
 The current WG-Draft version is built automatically from the master branch and can be accessed at: https://openid.github.io/OpenID4VP/openid-4-verifiable-presentations-wg-draft.html
 
-### Running Tests ###
```

Comment:
I am not ok removing these without adding a URL or something to the conformance tests or something equivalent.

Reply:
I'm happy to add a link to conformance tests and have done so, thanks!

Just to clarify (and I've updated the main description of the PR to say this now), these tests test the examples in the specification against a schema that's not up to date, so the tests fails. Keeping an incorrect & unmaintained schema in the repo is just very unhelpful to anyone looking at the repo.

I'm making no judgement on the value of these things and would be very happy if someone wants to raise a new PR that puts back / updates the schema and adds running the tests into the GitHub actions.